app/src/main/res/layout/fragment_main_dialog.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <EditText
+        android:id="@+id/parking_available_edit_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:inputType="text"
```

Comment:
instead of text, I think that should be a number

Reply:
im using as string to save the parking size, so i thought it would be better to use a text

Reply:
the editText always will return a string but with this property you will set the keyboard type, and as you are waiting numbers, not letters, the best will be to configure the keyboard in that way

Reply:
done